scrapper.py

```diff
@@ -1,6 +1,17 @@
 """
 Crawler implementation
 """
+import random
```

Comment:
1. imports in alphabetical order
2. imports in following order: Python standard library, empty line, external libraries, empty line, local modules